grails-app/services/org/broadinstitute/orsp/QueryService.groovy

```diff
@@ -978,7 +983,8 @@ class QueryService implements Status {
                             commentMap.get(issue.projectKey)?.any{it.description?.toLowerCase()?.contains(freeText)} ||
                             eventMap.get(issue.projectKey)?.any{it.summary?.toLowerCase()?.contains(freeText)} ||
                             issue.summary?.toLowerCase()?.contains(freeText) ||
-                            issue.description?.toLowerCase()?.contains(freeText)
+                            issue.description?.toLowerCase()?.contains(freeText) ||
+                            issue.projectKey?.toString()?.toLowerCase()?.equals(freeText)
```

Comment:
This looks like the only change required 👍 
However, I think it should follow the same free-text pattern of contains rather than equals.